### PR TITLE
Test UI components

### PR DESCRIPTION
**Pull request by thani-sh**

### CODE REVIEW

File: src/client/ui/controls.js [hunk 1]
```diff
@@ -180,7 +182,7 @@ export default class StorybookControls extends React.Component {
             onClear={this.clearFilterText.bind(this)}
           />
         </div>
-        <div style={listStyle}>
+        <div style={listStyle} className="stories-list">
```
Comment (arunoda) [inline src/client/ui/controls.js]: Do not use classNames just for the sake of testing.
May be we can simply get the html `.html()` with enzyme and do some regexp search for what you want to search.

Comment (arunoda) [inline src/client/ui/controls.js]: If there is no other option, use this.

**Comment by mnmtanish** [comment]
Removed classes

**Comment by arunoda** [comment]
Great.

**Pull request merged**